packages/ingest/src/S3ProviderClient.ts

```diff
@@ -81,6 +82,7 @@ class S3ProviderClient implements ProviderClient {
    *
    * @param {Object} params - the full path to the remote file to be fetched
    * @param {string} params.sourceKey - the full path to the remote file to be fetched
+   * @param {string} params.ACL
```

Comment:
Specify correct type in JSDocs, and indicate that it's optional with `[]`

```suggestion
   * @param {ObjectCannedACL} [params.ACL]
```